### PR TITLE
My Recipe App: Setup database, views and models

### PR DESCRIPTION
**Pull request by Mutalenic**
## In this milestone we;

 - Set up controllers
 - Set up the database
- Set up view
 - Set up model 
 - Ran migrations
 - Fixed linter errors

### CODE REVIEW

File: spec/requests/users_spec.rb [hunk 1]
```diff
@@ -0,0 +1,130 @@
+require 'rails_helper'
+
```
Comment (Paskab012) [inline spec/requests/users_spec.rb:2]: While running the test, it shows that the `rails_helper` can not be
 loaded. Would you mind if you take a look at the problem and fix it??? Or if your tests passed well on your laptop, Kindly include the screenshot and a detailed way of testing the app into the Pull Request and go-ahead. Request another review. Happy coding 👏🏽 

Screen Shot 2022-07-06 at 06 21 35

**Review by Paskab012 — CHANGES_REQUESTED** [review]
Hello @Mutalenic , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Just a very quick adjustment before moving  forward.

## To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: spec/requests/users_spec.rb (hunk 1 above)
Comment (Mutalenic) [reply spec/requests/users_spec.rb:2]: Hi@ Pascal, can you give suggestions on how we can resolve this

**Review by Paskab012 — APPROVED** [review]
Hello @Mutalenic , 

## STATUS APPROVED 🟢  💯 👌🏽 
There nothing eslse to add to the project, everything is working fine. Kindly go ahead a merge 👍🏽 it.
## To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 
- Approved ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Rolandm99 in your question so I can receive the notification.

**Pull request merged**